Assets/Scripts/LevelOverController.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LevelOverController : MonoBehaviour
+{
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
```

Comment:
again repeatition of logic which can be implemented in PlayerController. No need to create separate scripts for such small uses.